enterprise-suite/Makefile

```diff
@@ -35,26 +34,20 @@ script-tests:
 	$(CHART_DIR)/tests/lib/bats/bats $(CHART_DIR)/tests
 	python $(CHART_DIR)/scripts/lbc_test.py
 
-minikube-tests: build install-local
-	@echo Installed Console from helm charts: $(CHART)
-	@echo Running tests
-	( cd $(CHART_DIR)/tests ; ./smoketest ; )
+minikube-backend-tests: build
+	@echo Running minikube tests
+	( cd $(CHART_DIR)/tests ; ./smoketest_minikube.sh ; )
 
-backend-tests: script-tests minikube-tests
+openshift-backend-tests: build
```

Comment:
These new targets should be added to the `.PHONY` line at the bottom.  (And the old ones removed.)

I've been considering changing to the style that adds the `.PHONY` bit with each such target.  I think this would reduce the probability of these getting overlooked.  e.g.
```
[...]
.PHONY: openshift-backend-tests
openshift-backend-tests: build
[...]
```

Reply:
It's a matter of style I guess. Some people might find it annoying having .PHONY everywhere.

Reply:
I'll move it to one per target, because right now it's a mess of non-existent targets.